account_analytic_tag/views/account_analytic_tag_views.xml

```diff
@@ -0,0 +1,77 @@
+        <field name="model">account.analytic.tag</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="display_name" invisible="1" />
+                <field name="account_analytic_id" />
+                <field name="color" widget="color_picker" />
+                <field name="company_id" groups="base.group_multi_company" />
+            </tree>
+        </field>
+    </record>
+    <record id="account_analytic_tag_form_view" model="ir.ui.view">
+        <field name="name">account.analytic.tag.form</field>
+        <field name="model">account.analytic.tag</field>
+        <field name="arch" type="xml">
+            <form string="Analytic Tags">
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                    <group>
+                        <group>
+                            <field name="active" invisible="1" />
+                            <field name="name" />
+                            <field name="color" widget="color_picker" />
+                        </group>
+                        <group>
+                            <field name="account_analytic_id" />
+                            <field
+                                name="company_id"
+                                groups="base.group_multi_company"
+                            />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="account_analytic_tag_view_search" model="ir.ui.view">
+        <field name="name">account.analytic.tag.view.search</field>
+        <field name="model">account.analytic.tag</field>
+        <field name="arch" type="xml">
+            <search string="Search Analytic Tags">
+                <field name="name" />
+                <separator />
+                <filter
+                    string="Archived"
+                    name="inactive"
+                    domain="[('active', '=', False)]"
+                />
+            </search>
+        </field>
+    </record>
+    <record id="account_analytic_tag_action" model="ir.actions.act_window">
+        <field name="name">Analytic Tags</field>
+        <field name="res_model">account.analytic.tag</field>
+        <field name="view_mode">tree,form</field>
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+                Add a new tag
+            </p>
+        </field>
+    </record>
+    <menuitem
+        id="account_analytic_tag_menu"
+        name="Analytic Tags"
+        parent="account.menu_analytic_accounting"
+        action="account_analytic_tag_action"
+        groups="group_analytic_tags"
```

Comment:
Can we make this menu visible to the `account.group_account_manager` group instead. 

So accounting administrators can configure tags, but only users in the `account_analytic_tag.group_analytic_tags` can use them. This is important so we can use analytic tags just to categorize analytic accounts (in #585) but not show analytic tags on invoice/move lines.